packages/runtime/container-runtime/src/test/dataStoreContext.spec.ts

```diff
@@ -624,7 +628,7 @@ describe("Data Store Context Tests", () => {
             it("can successfully update referenced state", () => {
                 dataStoreAttributes = {
                     pkg: "TestDataStore1",
-                    snapshotFormatVersion: undefined,
+                    summaryFormatVersion: undefined,
```

Comment:
We should add tests with `snapshotFormatVersion` and make sure we can read that and still write out `summaryFormatVersion`.

Reply:
Added some tests. Let me know if this is not what you're looking for.